### PR TITLE
Updated update/reboot strategies page

### LINKED ISSUE #611: todo: clarify what is "update strategy" and "reboot strategy"
https://coreos.com/os/docs/latest/update-strategies.html - clarify what is "update strategy" and "reboot strategy". How to disable update, reboot, etc. Users mix up these terms.

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski 

resolves #611

### CODE REVIEW

File: os/update-strategies.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Update Strategies
+# Reboot Strategies on Updates
 
 The overarching goal of CoreOS is to secure the Internet's backend infrastructure. We believe that [automatically updating]({{site.baseurl}}/using-coreos/updates) the operating system is one of the best tools to achieve this goal.
 
 We realize that each CoreOS cluster has a unique tolerance for risk and the operational needs of your applications are complex. In order to meet everyone's needs, there are four update strategies that we have developed based on feedback during our alpha period.
 
-It's important to note that updates are always downloaded to the passive partition when they become available. A reboot is the last step of the update, where the active and passive partitions are swapped. These strategies control how that reboot occurs:
+It's important to note that updates are always downloaded to the passive partition when they become available. A reboot is the last step of the update, where the active and passive partitions are swapped (click [here][rollback] to know how to rollback update). These strategies control how that reboot occurs:
```
Comment (robszumski) [inline os/update-strategies.md]: instead of the click here syntax, can we just say "(rollback instructions)"

Comment (kayrus) [inline os/update-strategies.md]: updated

File: os/update-strategies.md [hunk 2]
```diff
@@ -73,7 +73,19 @@ The `off` strategy is also straightforward. The update will be installed onto th
...
+An easy solution to this problem is to use iPXE and reference images [directly from the CoreOS storage site]({{site.baseurl}}/os/docs/latest/booting-with-ipxe.html#setting-up-the-boot-script). The `alpha` URL is automatically pointed to the new version of CoreOS as it is released.
+
+## Disable Automatic Updates Daemon
+
+In case when you don't want to install updates onto the passive partition and avoid update process on failure reboot, you can disable `update-engine` service manually with `sudo systemctl stop update-engine` command (it will be enabled automatically next reboot). Or if you wish to disable automatic updates permanently, use Cloud-Config:
+
+```yaml
+#cloud-config
+coreos:
+    units:
```
Comment (crawford) [inline os/update-strategies.md]: Indent less.

File: os/update-strategies.md [hunk 3]
```diff
@@ -209,3 +221,5 @@ write_files:
         fi
         exit 0
 ```
+
+[rollback]: /os/docs/latest/manual-rollbacks.html
```
Comment (crawford) [inline os/update-strategies.md]: .md

Comment (kayrus) [inline os/update-strategies.md]: .md doesn't work https://coreos.com/os/docs/latest/manual-rollbacks.md

Comment (crawford) [inline os/update-strategies.md]: These files get rewritten, replacing the .md with .html. The markdown should be valid on it's own.

**Comment by kayrus** [comment]
@crawford Fixed indents and link.

**Comment by crawford** [comment]
Get rid of the template in the URLs. This is rewritten automatically. The docs should be valid in source-form first.

**Comment by kayrus** [comment]
Removed templates

**Comment by crawford** [comment]
The links are not valid.

**Comment by kayrus** [comment]
Fixed links. Now they should work.

**Comment by crawford** [comment]
LGTM

**Pull request merged**